deploy/operator/ztp/assisted-installer-crds-playbook.yaml

```diff
@@ -1,55 +1,182 @@
-    - mac_address: "{{ lookup('env', 'MAC_ADDRESS') }}"
-    - encoded_username: "{{ lookup('env', 'ENCODED_USERNAME') }}"
-    - encoded_password: "{{ lookup('env', 'ENCODED_PASSWORD') }}"
-    - address: "{{ lookup('env', 'ADDRESS') }}"
-    - cluster_subnet: "{{ lookup('env', 'CLUSTER_SUBNET') }}"
-    - cluster_host_prefix: "{{ lookup('env', 'CLUSTER_HOST_PREFIX') }}"
-    - external_subnet: "{{ lookup('env', 'EXTERNAL_SUBNET') }}"
-    - service_subnet: "{{ lookup('env', 'SERVICE_SUBNET') }}"
+    extra_baremetalhosts: "{{ lookup('env', 'EXTRA_BAREMETALHOSTS_FILE') | default('/home/test/dev-scripts/ocp/ostest/extra_baremetalhosts.json') }}"
+    spoke_namespace: "{{ lookup('env', 'SPOKE_NAMESPACE') | default('assisted-spoke-cluster') }}"
+    cluster_name: "{{ lookup('env', 'ASSISTED_CLUSTER_NAME') | default('assisted-agent-cluster-install') }}"
+    cluster_image_set_name: "{{ lookup('env', 'ASSISTED_OPENSHIFT_VERSION') }}"
+    cluster_release_image: "{{ lookup('env', 'ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE') }}"
+    cluster_deployment_name: "{{ lookup('env', 'ASSISTED_CLUSTER_DEPLOYMENT_NAME') | default('assisted-test-cluster') }}"
+    infraenv_name: "{{ lookup('env', 'ASSISTED_INFRAENV_NAME') | default('assisted-infra-env') }}"
+    agent_cluster_install_name: "{{ lookup('env', 'ASSISTED_AGENT_CLUSTER_INSTALL_NAME') | default('assisted-agent-cluster-install') }}"
+    pull_secret_name: "{{ lookup('env', 'ASSISTED_PULLSECRET_NAME') | default('assisted-pull-secret') }}"
 
+    ssh_private_key_name: "{{ lookup('env', 'ASSISTED_PRIVATEKEY_NAME') }}"
+    ssh_private_key_file: "{{ lookup('env', 'ASSISTED_PRIVATEKEY_FILE') | default('/root/.ssh/id_rsa') }}"
+    ssh_public_key_file: "{{ lookup('env', 'ASSISTED_PRIVATEKEY_FILE') | default('/root/.ssh/id_rsa.pub') }}"
+    ssh_public_key: "{{ lookup('file', ssh_public_key_file) }}"
+
+    cluster_subnet: "{{ lookup('env', 'CLUSTER_SUBNET') }}"
+    cluster_host_prefix: "{{ lookup('env', 'CLUSTER_HOST_PREFIX') }}"
+    external_subnet: "{{ lookup('env', 'EXTERNAL_SUBNET') }}"
+    service_subnet: "{{ lookup('env', 'SERVICE_SUBNET') }}"
   tasks:
-  - name: create directory for generated CRDs
-    file:
-      name: generated
-      state: directory
-  - name: write the cluster image set crd
-    template:
-      src: "clusterImageSet.j2"
-      dest: "generated/clusterImageSet.yaml"
-
-  - name: write the infraEnv crd
-    template:
-      src: "infraEnv.j2"
-      dest: "generated/infraEnv.yaml"
-
-  - name: write the clusterDeployment crd
-    template:
-      src: "clusterDeployment.j2"
-      dest: "generated/clusterDeployment.yaml"
-
-  - name: write the agentClusterInstall crd
-    template:
-      src: "agentClusterInstall.j2"
-      dest: "generated/agentClusterInstall.yaml"
-
-  - name: write the baremetalHost crd
-    template:
-      src: "baremetalHost.j2"
-      dest: "generated/baremetalHost.yaml"
+  - name: Create spoke namespace
+    community.kubernetes.k8s:
```

Comment:
Got below error- 
`ERROR! couldn't resolve module/action 'community.kubernetes.k8s'. This often indicates a misspelling, missing collection, or incorrect module path.`

Ref: https://github.com/ansible-collections/kubernetes.core/issues/159